src/security/PeerOptions.cc

```diff
@@ -297,130 +297,130 @@ static struct ssl_option {
 
```

Comment:
Please sync the ssl_option `value` type with OpenSSL v3 [changes](https://github.com/openssl/openssl/commit/f04bb0bce490de847ed0482b8ec9eabedd173852) to become uint64_t instead of long. Please try to avoid duplicating that code/type in Squid code -- either name it (e.g., ssl_option::Value) or, if there are only a couple of occurrences in low-level code, try using `decltype` for the second occurrence.

Reply:
Done in 3815b94d14858c503ad67adcaec8d98b1c1d5b24